File: CodeForces/Java/CF_282A_Bit++.java
//  Problem: 282A. Bit++
//  Link: https://codeforces.com/problemset/problem/282/A

import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner scan = new Scanner(System.in);
    int result = 0;
    int count = scan.nextInt();
    for ( int i = 0; i < count; i++) {
      String statement = scan.next();
      if (statement.charAt(1) == '+'){
        result ++;
      }
      if (statement.charAt(1) == '-'){
        result --;
      }
    }
    scan.close();
    System.out.println(result);
  }
}

File: CodeForces/Python/CF_282A_Bit++.py
# Problem: 282A. Bit++
# Link: https://codeforces.com/problemset/problem/282/A

count = int(input())
result = 0
for i in range(count):
  statement = input()
  if (statement[1]=='-'):
    result -=1
  elif (statement[1]=='+'):
    result +=1
print(result)